pydantic/fields.py

```diff
@@ -1,4 +1,5 @@
 import warnings
+from collections.abc import Callable
```

Comment:
This should be `typing.Callable`

Reply:
Something changes in the `typing` module between Python 3.6 and 3.7. Unfortunately, it can be both.